### ISSUE TITLE
Use adaptive design

### ISSUE THREAD

**Issue by chaabaj** [codeforseoul/-deprecated-showcase#2]
By using adaptive design we can provide a more convenient view for each device(mobile, tablets, desktop). Bootstrap and responsive design sometimes they are not enough and sometimes you want to create a specific view and adapted to the device. Some plugin for AngularJS provide a way to handle this : https://github.com/ghengeveld/angular-adaptive-templating. So we can use bootstrap with adapted views. So what about responsive + adaptive design ?

We can provide adapted view in the server also in the server side

**Comment by hoony-o-1** [comment]
@corneadoug was negative for responsive design at the beginning. I agree with his idea. I think it's better to build a prototype as fast as possible at the beginning. and we can extend adaptive designs on next steps or build a mobile app.

**Comment by chaabaj** [comment]
Yes sure :+1: We can for the beginning just provide desktop view and when we finished provide mobile view :)

**Comment by corneadoug** [comment]
The plugin looks interesting, especially for separating your html and maintaining it.
The only problem would be duplicate of code between views, and changing all of them when we change one size.

I think AngularJS has enough tool to handle responsive or conditional views by default (ng-class, ng-if, mediaqueries in css)

But we can always give it a try later

**Comment by chaabaj** [comment]
Yes but instead of ng-if it make it once by just redirect to the correct view. For duplicate code we can make html blocks using ng-include or directive for more complex block.

Like this : 

someFeature/
    blocks/
          someblock.tpl.html
    desktop.tpl.html
    mobile.tpl.html

but let see it later :+1:
